app/views/hud_reports/_parameters.haml

```diff
@@ -6,15 +6,18 @@
       %label.label.label-default.parameter-label Projects
       %label.label.label-primary.parameter-value
         %strong
-          - value.compact.delete_if(&:blank?).each do |project_id|
+          - compacted = value.compact.delete_if(&:blank?)
+          - compacted[0..4].each do |project_id|
             - project = GrdaWarehouse::Hud::Project.find(project_id)
             = link_to_if can_view_projects?, project.name, project_path(project_id)
             %br
+          - if compacted.size > 5
+            #{compacted.size - 5} more...
```

Comment:
Nice!

Reply:
I wonder if we should throw a tooltip on there with the remainder, or the whole set....